dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -1941,24 +1941,24 @@ private long getContentSelectedLanguageId(HttpServletRequest request){
 				List<Map<String, Object>> children = new ArrayList<Map<String,Object>>();
 
 				List<Folder> subFolders = folderAPI.findSubFolders(host,user,false);
-				for (Folder f : subFolders) {
+				for (final Folder folder : subFolders) {
 
 						List permissions = new ArrayList();
 						try {
-							permissions = permissionAPI.getPermissionIdsFromRoles(f, roles, user);
+							permissions = permissionAPI.getPermissionIdsFromRoles(folder, roles, user);
 						} catch (DotDataException e) {
 							Logger.error(this, "Could not load permissions : ",e);
```

Comment:
Let's log the ID of the folder here too.

Reply:
Done